CPT331.iOS/CPT331.iOS/CrimeManager.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  CrimeManager.swift
+//  CPT331.iOS
+//
+//  Created by Peter Weller on 13/10/2016.
+//  Copyright © 2016 Peter Weller. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+import Alamofire
+import SwiftyJSON
+
+/// Handles retrieval and parsing of event data from the EventGuardian JSON API
+class CrimeManager: JSONAPI {
+    static let sharedInstance = CrimeManager()
+    
+    // TODO: move domain to info.plist
+    /// EventGuardian API endpoint for crime data (read-only)
+    private let ENDPOINT = "http://ec2-52-32-105-85.us-west-2.compute.amazonaws.com/api/Crime/CrimesByCoordinate"
```

Comment:
I think we should move at least up to the host name into a configuration file.

eg ```ENDPOINT = hypotheticalConfigurationClass.EventGuardianApi + "/api/Crime/CrimesByCoordindate"```

This is for two reasons:
 1. It allows us to easily change the production environment without having to update and recompile code. The fact is that the hostname ```ec2-52-32-105-85.us-west-2.compute.amazonaws.com``` is it self a temporary site.
 2. It allows the implementation of staged deployments. 
   * You could deploy the API and App to a staging environment by transforming the configs for staging.
   * UAT then takes place on staging.
   * Then the API and App are promoted to production, driven by another config transform.
   * The key point is there were no code changes between staging/production. Furthermore, the API / App wont even require recompiling.

I think configuration should be easily achievable within the Swift framework.